internal/k8sinternal/client.go

```diff
@@ -85,6 +85,7 @@ func IsRunningInCluster(client Client) bool {
 type ClientOptions struct {
 	// Namespace filters resources by namespace. Defaults to all namespaces.
 	Namespace string
+	IncludeGenerated bool
```

Comment:
Could you add a comment here explaining what the field does so it'll end up in the Go docs?